doc/source/news.rst

```diff
@@ -7,6 +7,36 @@
 News
 ====
 
+.. _release-9-1-1:
+
+Release 9.1.1 - 2019-12-27
+--------------------------
+
+Improvements
+^^^^^^^^^^^^
+
+* [:doc:`/reference/commands/load`] Added support Apache Arrow format data.
```

Comment:
"Added support for XXX"

Reply:
Thank you for your comment.
I've fixed it.